Mathlib/Topology/Algebra/SeparationQuotient.lean

```diff
@@ -391,6 +409,14 @@ theorem mk_outCLM (x : SeparationQuotient E) : mk (outCLM K E x) = x :=
 @[simp]
 theorem mk_comp_outCLM : mk ∘ outCLM K E = id := funext (mk_outCLM K)
 
+variable {K} in
+theorem postcomp_mkCLM_surjective {L : Type*} [Semiring L] (σ : L →+* K)
+    (F : Type*) [AddCommMonoid F] [Module L F] [TopologicalSpace F] :
```

Comment:
The variable order looks a little strange here; I'd expect `F` to come before `σ`, but maybe there's a good reason not to do that.

Reply:
My reason was: in "`σ`-semilinear maps from `E` to `F`", `σ` comes before `F`.